docs/sql-programming-guide.md

```diff
@@ -1061,7 +1061,7 @@ the following case-sensitive options:
   </tr>
 
   <tr>
-    <td><code>partitionColumn, lowerBound, upperBound, numPartitions</code></td>
```

Comment:
We should revert it back. These four parameters are related. 

`These options must all be specified if any of them is specified`